### ISSUE TITLE
Fix accessibility issues

### ISSUE THREAD

**Issue by maxrothman** [eepMoody/open5e#156]
Ran aXe, got the following issues:
- contrast is too low between color of <a> elements and background colors. Should have contrast ratio of at least 3.96.
- search box form should have a label (aria-label, aria-labelledby, wrapping <label>, or title attribute)
- table headers should have scope attribute

Might also want to check on http://www.evaluera.co.uk/
